README.md

```diff
@@ -110,7 +110,7 @@ Version 5.0 (Beta)
 -----------
 
 Turi Create 5.0 -- now in beta -- includes:
-* GPU Acceleration on Macs (10.14+)
+* GPU Acceleration on Macs for Image Similarity (10.13+), Image Classification (10.13+), Object Detection (10.14+), and Activity Classification (10.14+).
```

Comment:
Is it clear this is a macOS version?

Reply:
Maybe not, but it's starting to get really verbose. Can you think of a better way to express this nuance?